src/app/app-routing.module.ts

```diff
@@ -0,0 +1,24 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+
+const routes: Routes = [
+  {
+    path: 'login',
+    loadChildren: './login/login.module#LoginModule'
+  },
+  {
+    path: 'shell',
+    loadChildren: './shell/shell.module#ShellModule'
+  },
+  {
+    path: '',
```

Comment:
try to make shell component as root route

Reply:
Done!